### ISSUE TITLE
aws_security_group throws an error on action :delete (No resource, method, or local variable named `existing_vpc')

### ISSUE THREAD

**Issue by vinyar** [chef/chef-provisioning-aws#92]
create recipe:
```ruby
aws_security_group "#{name}-ssh" do
  inbound_rules [{:ports => 22, :protocol => :tcp, :sources => ['0.0.0.0/0'],:subnet => "vpc-0eeb366b"}]
end
```

remove recipe:
```ruby
aws_security_group "#{name}-ssh" do
  action 'delete'
end
```

error:
```ruby
    Error executing action `delete` on resource 'aws_security_group[alexv-ssh]'
===
    NameError
    No resource, method, or local variable named `existing_vpc' for `Chef::Provider::AwsSecurityGroup ""'
```

Full error text:
```ruby
$ chef-client -r 'recipe[webserver::deprovision]' -z
[2015-02-08T09:26:29-08:00] INFO: Started chef-zero at http://localhost:8889 with repository at /Users/alexvinyar/Documents/Projects/chef-provisioning
  One version per cookbook

[2015-02-08T09:26:29-08:00] INFO: Forking chef instance to converge...
Starting Chef Client, version 12.0.3
[2015-02-08T09:26:29-08:00] INFO: *** Chef 12.0.3 ***
[2015-02-08T09:26:29-08:00] INFO: Chef-client pid: 74369
[2015-02-08T09:26:31-08:00] INFO: Setting the run_list to [#<Chef::RunList::RunListItem:0x007f8ccb052850 @version=nil, @type=:recipe, @name="webserver::deprovision">] from CLI options
[2015-02-08T09:26:31-08:00] INFO: Run List is [recipe[webserver::deprovision]]
[2015-02-08T09:26:31-08:00] INFO: Run List expands to [webserver::deprovision]
[2015-02-08T09:26:31-08:00] INFO: Starting Chef Run for alexv
[2015-02-08T09:26:31-08:00] INFO: Running start handlers
[2015-02-08T09:26:31-08:00] INFO: Start handlers complete.
[2015-02-08T09:26:31-08:00] INFO: HTTP Request Returned 404 Not Found : Object not found: /reports/nodes/alexv/runs
resolving cookbooks for run list: ["webserver::deprovision"]
[2015-02-08T09:26:32-08:00] INFO: Loading cookbooks [webserver@0.1.0]
Synchronizing Cookbooks:
[2015-02-08T09:26:32-08:00] INFO: Storing updated cookbooks/webserver/recipes/deprovision.rb in the cache.
  - webserver
Compiling Cookbooks...
Converging 3 resources
Recipe: webserver::deprovision
  * aws_security_group[alexv-ssh] action delete[2015-02-08T09:26:35-08:00] INFO: Processing aws_security_group[alexv-ssh] action delete (webserver::deprovision line 12)


    ================================================================================
    Error executing action `delete` on resource 'aws_security_group[alexv-ssh]'
    ================================================================================

    NameError
    ---------
    No resource, method, or local variable named `existing_vpc' for `Chef::Provider::AwsSecurityGroup ""'

    Resource Declaration:
    ---------------------
    # In /Users/alexvinyar/Documents/Projects/chef-provisioning/.chef/local-mode-cache/cache/cookbooks/webserver/recipes/deprovision.rb

     12: aws_security_group "#{name}-ssh" do
     13:   # inbound_rules [{:ports => 22, :protocol => :tcp, :sources => ['0.0.0.0/0'],:subnet => "vpc-0eeb366b"}]
     14:   action 'delete'
     15: end
     16:

    Compiled Resource:
    ------------------
    # Declared in /Users/alexvinyar/Documents/Projects/chef-provisioning/.chef/local-mode-cache/cache/cookbooks/webserver/recipes/deprovision.rb:12:in `from_file'

    aws_security_group("alexv-ssh") do
      action [:delete]
      retries 0
      retry_delay 2
      default_guard_interpreter :default
      id "alexv-ssh"
      security_group_id "sg-ba47e0df"
      declared_type :aws_security_group
      cookbook_name "webserver"
      recipe_name "deprovision"
    end

[2015-02-08T09:26:35-08:00] INFO: Running queued delayed notifications before re-raising exception

Running handlers:
[2015-02-08T09:26:35-08:00] ERROR: Running exception handlers
Running handlers complete
[2015-02-08T09:26:35-08:00] ERROR: Exception handlers complete
[2015-02-08T09:26:35-08:00] FATAL: Stacktrace dumped to /Users/alexvinyar/Documents/Projects/chef-provisioning/.chef/local-mode-cache/cache/chef-stacktrace.out
Chef Client failed. 0 resources updated in 6.030797 seconds
[2015-02-08T09:26:35-08:00] ERROR: aws_security_group[alexv-ssh] (webserver::deprovision line 12) had an error: NameError: No resource, method, or local variable named `existing_vpc' for `Chef::Provider::AwsSecurityGroup ""'
[2015-02-08T09:26:36-08:00] FATAL: Chef::Exceptions::ChildConvergeError: Chef run process exited unsuccessfully (exit code 1)
```

**Comment by christinedraper** [comment]
Same as issue #74 - fix is in #91 pull request

**Comment by vinyar** [comment]
sweetness, thanks. I need to learn to search better.
